code/modules/mob/living/simple_animal/friendly/cockroach.dm

```diff
@@ -32,12 +32,13 @@
 	if(ismob(AM))
 		if(isliving(AM))
 			var/mob/living/A = AM
-			if(A.mob_size > MOB_SIZE_TINY)
-				if(prob(squish_chance))
-					A.visible_message("<span class='notice'>\The [A] squashed \the [name].</span>", "<span class='notice'>You squashed \the [name].</span>")
-					death()
-				else
-					visible_message("<span class='notice'>\The [name] avoids getting crushed.</span>")
+			if(!isdrone(A))
+				if(A.mob_size > MOB_SIZE_TINY)
```

Comment:
Why not just set drones mob size to tiny? I don't think they're supposed to be very big "lore"-wise and mob size is used almost nowhere else (I think that would let drones get under plastic flaps but again, that makes some level of sense).